tests/OSPSuite.Presentation.Tests/Services/DataImporterSpecs.cs

```diff
@@ -226,6 +234,29 @@ public void should_convert_MW_correctly_excel()
          result[0].AllButBaseGridAsArray[0].DataInfo.MolWeight.ShouldBeEqualTo(2.08E-07);
       }
 
+      [Observation]
+      public void should_not_allow_import_when_checking_MW_against_molecule()
```

Comment:
I think the observation does a lot of the because. Normally you only observe in the observation block and set the stuff up before. Fine for now, but I'd like to discuss how to organize test a bit more in the AAA style (arrange, act, assert)

Reply:
Sounds actually like a great discussion to have on monday! always up to improving the way I write code! :D